src/commands/dev/list.ts

```diff
@@ -57,7 +59,11 @@ export default class DevList extends BaseCommand {
         };
       }
     }
-    this.log(JSON.stringify(output, null, 2));
+    if (!Object.keys(output).length) {
+      this.log('There are no active dev instances yet. Use `architect dev` to create one.');
+    } else {
+      this.log(JSON.stringify(output, null, 2));
+    }
```

Comment:
I know this is already merged but we may want to change this back - if I specifically want the output format to be json, we shouldn't return non-json even if there's no results (returning `{}` for example would at least not break somebodys `JSON.parse()` equivalent or pipe-to-jq)